sig-instrumentation/charter.md

```diff
@@ -0,0 +1,73 @@
+# SIG Instrumentation Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Owns best practices for cluster observability through metrics and logging
+across all Kubernetes components and development of components required for all
+Kubernetes clusters (eg. heapster, metrics-server).
+
+### In scope
+
+SIG-Instrumentation revolves around the process of instrumenting and exposing
+observability signals.
+
+The act of instrumenting components not owned by SIG-Instrumentation is out of
+scope, however SIG-Instrumentation is there to advise any contributors with
+instrumentation decisions.
+
+As well as giving advice in regards to instrumentation, SIG-Instrumentation
+coordinates metric requirements of different SIGs for other components through
+finding common APIs (such as the resource/core, custom and external metrics
+APIs).
+
+SIG Instrumentation owns the interface of the log file and directory structure
```

Comment:
we did not discuss this in the sig-node charter. 

i think this should have some further refinement.

at best I think this is with sig-node, and not exclusively instrumentation.  the sig-node charter says "Issues related to node, pod, container monitoring (with [sig-instrumentation](../sig-instrumentation))" and probably should have also had a line for logging w/ sig-instrumentation.  I think this is shared responsibility across the two sigs.

Reply:
Makes sense. Will move that to shared responsibilities.

Reply:
I deleted this line and added: "The interface of log files and their directory structure written out by container runtimes to be processed by other systems further, is shared responsibility between [SIG Node](sig-node) and SIG Instrumentation." To the cross-cutting section.